### PR TITLE
Update databases.md

### PR DESCRIPTION
**Pull request by splitfinity**
Made a few changes:
- Added a link for SQL Managed Instance
- Updated link and description for synapse analytics
- Updated description for power bi

### CODE REVIEW

**Comment by PRMerger7** [comment]
@splitfinity : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/MicrosoftDocs/azure-docs 
 Private repo URL: https://github.com/MicrosoftDocs/azure-docs-pr 
 We have closed this pull request. Please resubmit the updates through the private repo. For more information, [see the contributor guide].(https://docs.microsoft.com/en-us/contribute/)

**Review by EdPrice-MSFT — DISMISSED** [review]
Fantastic updates!

**Review by EdPrice-MSFT — DISMISSED** [review]
Fantastic updates!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by PRMerger7** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @splitfinity, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/aws-professional/databases.md [hunk 1]
```diff
@@ -50,16 +53,16 @@ Costs for AWS RDS are determined by the amount of hardware resources that your i
 
 - [Data Lake Analytics](/azure/data-lake-analytics/data-lake-analytics-overview): large-scale analytics service optimized to work with Data Lake Store
 
-- [Power BI](https://powerbi.microsoft.com): used to power data visualization.
+- [Power BI](https://powerbi.microsoft.com): a business analytics service that provides the capabilities to creach rich interactive data visualizations.
```
Comment (ShannonLeavitt) [inline docs/aws-professional/databases.md:56]:
```suggestion
- [Power BI](https://powerbi.microsoft.com): a business analytics service that provides the capabilities to create rich interactive data visualizations.
```
Fixing typo

**Comment by akdataai** [comment]
Can I suggest adding specific Cosmos documentation to mirror the GCP offerings of BigTable, Firestore, Firebase. 

Cloud Bigtable - Cosmos DB Cassandra API
Cloud Firestore - Cosmos DB Mongo API
Cloud Firebase Realtime Database - Cosmos DB Change Feed

I'd also include Cosmos DB analytical store against Google Bigtable 

Are we missing Google Datastore, for which we would have Cosmos DB SQL API

**Review by akdataai — CHANGES_REQUESTED** [review]
Can we expand this to include Cosmos and all it's features including Cosmos Analytical Store and Change Feed.

Also references to Cortana AI Suite should be changed to Azure AI Platform?

**Review by EdPrice-MSFT — APPROVED** [review]
Good changes

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by PRMerger17** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @splitfinity, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by ktoliver** [comment]
I moved the commits to the private repo to take advantage of automated metadata and link validation. I'll close this pull request. The new PR will be merged soon.

**Pull request closed without merging**